src/test/java/com/duedil/mesos/UtilsTest.java

```diff
@@ -29,14 +30,52 @@ public void testExecutorEndpointThrowsExceptionWithNullInput() {
     @Test
     public void testGetEnvFetchesValueFromEnvironment() {
         String expected = "http://127.0.0.1:5050";
-        String actual = getEnv("MESOS_AGENT_ENDPOINT");
+        String actual = getEnvOrThrow("MESOS_AGENT_ENDPOINT");
         assertThat(actual, is(equalTo(expected)));
     }
 
     @SuppressWarnings("unused")
     @Test(expected = NullPointerException.class)
     public void testGetEnvThrowsNpeWhenEnvVarDoesntExist() {
-        String actual = getEnv("MESOS_FROBNITZER");
+        String actual = getEnvOrThrow("MESOS_FROBNITZER");
+    }
+
+    @Test
+    public void testTimeConversionHoldsAllUnits() {
+        TimeConversion tc = TimeConversion.getInstance();
+
+        long oneMili = 1;
```

Comment:
Yeah, I could make the ones in the class public and use them here. Will do that.